frontend/src/app/pages/events/id/events.component.css

```
@@ -0,0 +1,167 @@
+.events-container {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    min-height: 100dvh;
```

dvh is kacke entweder svh oder nur vh
```suggestion
    min-height: 100vh;
```